src/init/index.internals.js

```diff
@@ -112,7 +114,9 @@ module.exports = function(options, file, fileContent) {
 
     try {
       sources.map = JSON.parse(stripBom(fs.readFileSync(mapFile, 'utf8')));
-    } catch (e) {} //should we really swallow this error?
+    } catch (e) {
+      debug(function() { return 'warn: external source map not found or invalid: ' + mapFile; });
```

Comment:
Should we log anything from `e`? Should we log the stack trace? Otherwise LGTM

Reply:
Hmmm, my use case was a missing file, and the similar code in _fixSources couldn’t care less about e, but now when I look at JSON.parse and stripBom, it seems beneficial to include something out of the exception indeed. How would you propose to stringify it?

As for the stack, I think that would be overkill as the logger already has a location and it’s more or less narrow.